pikaday.js

```diff
@@ -1082,13 +1102,22 @@
             }
 
             for (var c = 0; c < opts.numberOfMonths; c++) {
-                randId = 'pika-title-' +
-                  Math.random().toString(36).replace(/[^a-z]+/g, '').substr(0, 2);
+                randId = 'pika-title-' + c;
+                var pikaDaysContainer = '<div id="pika-days-' + c + '">' + this.render(this.calendars[c].year, this.calendars[c].month, randId) + '</div>';
+                var pikaTitleContainer = renderTitle(this, c, this.calendars[c].year, this.calendars[c].month, this.calendars[0].year, randId);
 
-                html += '<div class="pika-lendar">' + renderTitle(this, c, this.calendars[c].year, this.calendars[c].month, this.calendars[0].year, randId) + this.render(this.calendars[c].year, this.calendars[c].month, randId) + '</div>';
+                // update the days container only
+                if (this.el.querySelector('div#pika-days-' + c)) {
```

Comment:
same enhancement we can do here, just store the `this.el.querySelector('div#pika-days-' + c)` in a variable and reuse it